pkg/ingress/model_build_listener_rules.go

```diff
@@ -249,3 +254,7 @@ func (t *defaultModelBuildTask) buildPathPatternCondition(_ context.Context, pat
 		},
 	}
 }
+
+func (t *defaultModelBuildTask) modelBuildListenerRuleTags(ctx context.Context) (map[string]string, error) {
```

Comment:
similar to listener tags, 

personally i prefer to generate listenerRule tags with tags on the single Ingresses requested for this listener rule.
what do you think?